src/Expectation.php

```diff
@@ -382,6 +402,20 @@ public function toHaveKey(string $key): Expectation
         return $this;
     }
 
+    /**
+     * Asserts that the value array has all of the provided $keys.
+     *
+     * @param array<mixed> $keys
```

Comment:
`@param array<int, int|string> $keys`.

Reply:
Of course!
Forgot about the tests, I'll get on it 😌